src/QMCWaveFunctions/VariableSet.h

```diff
@@ -171,11 +171,11 @@ struct VariableSet
     }
   }
 
-  inline void getParameterTypeList(std::vector<int>& types)
+  inline void getParameterTypeList(std::vector<int>& types) const
   {
-    std::vector<index_pair_type>::iterator PTit(ParameterType.begin()), PTend(ParameterType.end());
+    auto PTit(ParameterType.begin()), PTend(ParameterType.end());
```

Comment:
This file could really use an update to follow the naming conventions.
PTit, ParameterType, NameAndValue are actually all variables
pair_type, variable_type, etc are types.
It's painful to read.

Reply:
The VariableSet class ought to be replaced, or at least needs significant changes, as discussed in #3633 .   Updating the naming conventions as a separate step might not be worth the effort.